src/actions/actions.js

```diff
@@ -24,6 +23,7 @@ import {
 } from "./jsmd-selection";
 
 import evalQueue from "./evaluation-queue";
+import { postActionToEvalFrame } from "../port-to-eval-frame";
```

Comment:
This is not strictly true, actually - in this case, I actually need the `addToConsole` action thunk. Otherwise if I just use `disapatch({type:`ADD_TO_CONSOLE`, ...})` I don't get it passed through the action creator that creates the ID. Is there a different way of handling this? The eval frame actually creates the parts of the state and then forwards them back up to the editor, I believe.

Reply:
Open to try something that doesn't use `postActionToEvalFrame` here - do you have any suggestions about where to look in this case?